editor/editor_properties_array_dict.cpp

```diff
@@ -551,6 +554,13 @@ void EditorPropertyArray::_button_draw() {
 	}
 }
 
+void EditorPropertyArray::_button_add_item_draw() {
+	if (button_add_item && dropping) {
```

Comment:
`button_add_item` always exists here, no? It's where the `draw` signal comes from.

Reply:
True, this is just defensive programming. Want me to remove it?